docs/Update.md

```diff
@@ -1,27 +1,32 @@
 # How to update AzerothCore to the latest stable version
 
-## 1) Update your local sources
+## 1) Verify that you're on the latest release
+Check the [releases](https://github.com/azerothcore/azerothcore-wotlk/releases) page on the azerothcore-wotlk github page to see if there has been a new release since you last upgraded. If there has been a new release, follow the upgrade instructions to update to the latest release. **The next steps will not work properly unless you are on the latest release.**
```

Comment:
Nobody should look at releases as people will just download it. it is bad.

just link to this instead https://github.com/azerothcore/wiki/blob/master/docs/Upgrade-from-pre-3.0.0-to-latest-master.md

Reply:
I'll put that in, but what's our plan for the next time we release? I'm concerned that when we release 4.0.0, we'll forget to update this 'Update.md' page with a link to the 'Upgrade from pre-4.0.0 to latest master'

Reply:
Shin dont forget that, also that will be in a year or so